core/src/main/scala/stryker4s/extension/mutationtype/ConditionalExpression.scala

```diff
@@ -0,0 +1,13 @@
+package stryker4s.extension.mutationtype
+
+import scala.meta.Term
+import stryker4s.extension.TreeExtensions.IsConditionOfIf
+
+case object IfStatementCondition {
+
+  def unapply(term: Term): Option[(Term, Seq[BooleanLiteral])] = {
```

Comment:
Was there a specific reason this is done differently than the other mutators? Where the return is an `Option[Term]` (see boolean mutator). Then the function could be something like:

```scala
term.parent collect {
  case Term.If(cond, _, _) if cond == term => term
}
```

Reply:
This makes more sense to me too. 😅 👍